src/modules/esl-footnotes/core/esl-note-group.ts

```diff
@@ -4,17 +4,25 @@ import {ready} from '../../esl-utils/decorators/ready';
 
 import type {ESLNote} from './esl-note';
 
+interface Attribute {
+  name: string;
+  value: string;
+}
```

Comment:
Let's discuss changes in esl-note-group.ts today
Attribute propagation logic looks a little bit overcomplicated.

Reply:
You are right! It looks like an elephant on the bike. :)
I have already reworked esl-note-group.
What do you think about the new variant?